src/fabric_cli/commands/fs/ls/fab_fs_ls_item.py

```diff
@@ -21,9 +21,10 @@ def exec(workspace: Workspace, args):
         show_all or fab_state_config.get_config(fab_constant.FAB_SHOW_HIDDEN) == "true"
     )
 
-    utils_ui.print_output_format(
+    utils_ls.format_and_print_output(
+        sorted_elements_dict,
         args,
-        data=sorted_elements_dict,
-        hidden_data=VirtualItemContainerType if show_hidden else None,
-        show_headers=show_details,
+        show_details,
+        sorted_elements_dict[0].keys() if sorted_elements_dict else [],
```

Comment:
add the parameter name - like in previous implementation - hidden_data=....show_headers=.... It will make it more readable